commutatus/cm-nextjs-boilerplate
# feat: Add encrypted environment files system

## What?

Add commands to encrypt and decrypt staging and prod keys



## Testing?


1. Create `.env.local` and add test variable
2. Set `DECRYPTION_SECRET_KEY` in console
3. Run `encrypt-staging-env`
4. Run `dev:staging` and check if environment variable has the correct value

## Anything Else?

Readme needs to be updated to explain all the available features
